tfx_bsl/tfxio/parquet_tfxio_test.py

```diff
@@ -301,6 +301,34 @@ def _AssertFn(record_batch_list):
       record_batch_pcoll = (p | tfxio.BeamSource(batch_size=_NUM_ROWS))
       beam_testing_util.assert_that(record_batch_pcoll, _AssertFn)
 
+  def testOptionalColumnNames(self):
```

Comment:
thanks, could you also add a test when only a subset of columns is read

Reply:
it does, but through project path, which shouldn't be the default when one just wants to read subset of data (e.g. it's should not be necessary to have the whole schema). Users often take examples from tests, so seeing how it can be done here would be useful imo

Reply:
makes sense, I'll add it